FirebaseSessions/Sources/ApplicationInfo.swift

```diff
@@ -63,4 +75,12 @@ class ApplicationInfo: ApplicationInfoProtocol {
   var mccMNC: String {
     return FIRSESValidateMccMnc(networkInfo.mobileCountryCode, networkInfo.mobileNetworkCode) ?? ""
   }
+  
+  var environment: DevEnvironment {
+    let environment = envParams["FIREBASE_RUN_ENVIRONMENT"]
+    if (environment != nil) {
+      return DevEnvironment(rawValue: environment!.trimmingCharacters(in: .whitespaces).lowercased()) ?? DevEnvironment.prod
```

Comment:
Maybe in a follow up, could we add a log here or before logging to FireLog to tell us when we're using a non-prod environment?

Reply:
`if let environment = envParams....`

Reply:
Done. in the DevLogger.